config/default/kustomization.yaml

```diff
@@ -6,7 +6,7 @@
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: operator-
+namePrefix: eventing-
```

Comment:
With this change, our controller is now named `eventing-controller-manager` instead of `operator-controller-manager`